BPOR/BPOR.Rms/Views/Filter/Partials/_VolunteerInclusionExclusionCriteria.cshtml

```diff
@@ -7,58 +7,104 @@
                          )
 
-                @if (Model.ShowRecruitedFilter)
+            </div>
+
+            <input asp-for="ShowPreferredContactFilter" class="govuk-input" id="ShowPreferredContactFilter" value="@(Model.ShowPreferredContactFilter)" hidden />
+            <div class="govuk-form-group  @(Html.ViewData.ModelState["SelectedVolunteersPreferredContact"]?.Errors.Count > 0 ? "govuk-form-group--error" : "")">
+                @if (Model.ShowPreferredContactFilter)
                 {
-                    @Html.LabelFor(m => m.SelectedVolunteersRecruited, new { @class = "govuk-label exclusion_label" })
-                    <div id="volunteers-recruited-hint" class="govuk-hint">
-                        Do you want to find volunteers who have been recruited to this study?
+
+                    @Html.LabelFor(m => m.SelectedVolunteersPreferredContact, new { @class = "govuk-label exclusion_label" })
+                    <div id="volunteers-preferred-contact-hint" class="govuk-hint">
+                        Do you want to find volunteers who have a specific preferred contact method?
                     </div>
+
+                    @if (Html.ViewData.ModelState["SelectedVolunteersPreferredContact"]?.Errors.Count > 0)
+                    {
+                        <span id="SelectedVolunteersPreferredContact-error" class="govuk-error-message">
+                            <span class="govuk-visually-hidden">Error:</span>
+                            @Html.ValidationMessageFor(model => model.SelectedVolunteersPreferredContact)
+                        </span>
+                    }
+
                     @Html.DropDownListFor(
-                             m => m.SelectedVolunteersRecruited,
-                             Model.VolunteersRecruitedItems,
-                             new { @class = "govuk-select govuk-select-custom", ariaDescribedby = "volunteers-recruited-hint" }
-                             )
+                            m => m.SelectedVolunteersPreferredContact,
+                            Model.VolunteersPreferredContactItems,
+                            new { @class = "govuk-select govuk-select-custom", ariaDescribedby = "volunteers-preferred-contact-hint" }
+                            )
                 }
                 else
                 {
                     <div class="govuk-input_select__wrapper">
-                        <div class="govuk-label exclusion_label">Volunteers recruited</div>
+                        <div class="govuk-label exclusion_label">Volunteers preferred contact method</div>
                         <div class="govuk-hint govuk-!-margin-top-4" style="margin-bottom: 12px;">
-                            Study uses anonymous recruitment
+                            Studies recruiting anonymous participants can only contact volunteers by email.
                         </div>
-                    </div>            
+                    </div>
```

Comment:
when anonymous, filter should filter participants with Email contact preference. Currently filtering all